www/src/views/showcase/showcase-list.js

```diff
@@ -56,29 +55,27 @@ const ShowcaseList = ({ items, count }) => {
                 >
                   <ShowcaseItemCategories categories={node.categories} />
                 </div>
-                {node.source_url && (
-                  <div>
+                <div css={{flex: `0 0 auto`, textAlign: `right`}}>
+                  {node.source_url && (
                     <a
-                      css={{
-                        "&&": {
-                          color: colors.gray.bright,
-                          fontWeight: `normal`,
-                          borderBottom: `none`,
-                          boxShadow: `none`,
-                          "&:hover": {
-                            background: `none`,
-                            color: colors.gatsby,
-                          },
-                        },
-                      }}
+                      css={{...styles.shortcutIcon}}
                       href={node.source_url}
                       target="_blank"
                       rel="noopener noreferrer"
                     >
                       <GithubIcon style={{ verticalAlign: `text-top` }} />
                     </a>
-                  </div>
-                )}
+                  )}
+                  <a
+                    href={node.main_url}
+                    target="_blank"
+                    rel="noopener noreferrer"
+                    css={{...styles.shortcutIcon}}
+                  >
+                    {` `}
```

Comment:
Nitpicky alert, but we should move that blank `{` `}` in between the two `href`s/only render it when we have a link to the GitHub source.